modules/gcp/mysql/main.tf

```diff
@@ -43,7 +43,7 @@ data "google_compute_network" "my-network" {
 locals {
   ip_configuration = {
       ipv4_enabled = var.db_connectivity_type == "private" ? false : true
-      allocated_ip_range = var.db_connectivity_type == "private" ? "google-managed-services-${var.vpc_network_name}": null
+      allocated_ip_range = var.db_connectivity_type == "private" ? var.private_service_access_name : null
```

Comment:
Wouldn't this be a breaking change for existing `mysql`? Does it cause downtime?

Reply:
Yes, its a breaking change... Buy we are good for now since no one is using gcp for now

Reply:
Good to go then, just confirm by checking the db entries though.

Reply:
Checked and used by none.